tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -1178,6 +1213,139 @@ bool CorProfiler::ProfilerAssemblyIsLoadedIntoAppDomain(AppDomainID app_domain_i
            managed_profiler_loaded_app_domains.find(app_domain_id) != managed_profiler_loaded_app_domains.end();
 }
 
+HRESULT CorProfiler::RewriteForDistributedTracing(const ModuleMetadata& module_metadata, ModuleID module_id)
+{
+    HRESULT hr = S_OK;
+
+    if (IsDebugEnabled())
+    {
+        Logger::Info("pcbPublicKey: ", managed_profiler_assembly_property.pcbPublicKey);
+        Logger::Info("pcbPublicKey: ", HexStr(managed_profiler_assembly_property.ppbPublicKey,
+                                              managed_profiler_assembly_property.pcbPublicKey));
+        Logger::Info("pcbPublicKey: ");
```

Comment:
I think one or both of these messages should say `ppbPublicKey`? Not sure though 😄 
```suggestion
        Logger::Info("ppbPublicKey: ", HexStr(managed_profiler_assembly_property.ppbPublicKey,
                                              managed_profiler_assembly_property.pcbPublicKey));
        Logger::Info("ppbPublicKey: ");
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2020/commits/e1b26a25a9095a2a389a61a409e792c1b84a20b4